### DISCUSSION TITLE
Double ended range selector

### DISCUSSION THREAD

**Discussion by SergejGleithmann** [mdbootstrap/TW-Elements#2143] [Share ideas & request features]
I would find an extension of the default range selector handy in which one could both choose a maximum and a minimum value in the range.

**Comment by juujisai** [comment]
Hi, do you mean something like here in this example? 
https://tw-elements.com/docs/standard/forms/range/#min_and_max

**Reply by SergejGleithmann** [reply]
Yeah, I meant this element but with two draggable circles, so that you could choose two values with one range selector instead of one. Like this: 
![grafik](https://github.com/mdbootstrap/TW-Elements/assets/110982970/755d79cc-6447-461e-9729-196a8bcc37be)

**Reply by juujisai** [reply]
Okay I see. Soon we will add something like this:

https://mdbootstrap.com/docs/standard/forms/multi-range-slider/

**Reply by SergejGleithmann** [reply]
Ah, yes! That would be perfect! Thank your for the response!
